parsers/ajenti.py

```diff
@@ -18,29 +18,33 @@
 from signalr import Connection
 
 ZONE_PARAMS = {
+    # King's Island
+    # https://www.hydro.com.au/clean-energy/hybrid-energy-solutions/success-stories/king-island
     "AU-TAS-KI": {
         "hub": "TagHub",
         "method": "Dashboard",
         "tz": "Australia/Currie",
-        "source": "https://www.hydro.com.au/clean-energy/hybrid-energy-solutions/success-stories/king-island",  # Iframe: https://data.ajenti.com.au/KIREIP/index.html
+        "source": "hydro.com.au",  # Iframe: https://data.ajenti.com.au/KIREIP/index.html
     },
     # Flinders Island
     # https://github.com/electricitymaps/electricitymaps-contrib/issues/2533
     # https://en.wikipedia.org/wiki/Flinders_Island
+    # https://www.hydro.com.au/clean-energy/hybrid-energy-solutions/success-stories/flinders-island
     "AU-TAS-FI": {
         "hub": "flindershub",
         "method": "SendDashboard",
         "tz": "Australia/Hobart",
-        "source": "https://www.hydro.com.au/clean-energy/hybrid-energy-solutions/success-stories/flinders-island",
```

Comment:
You don't want to change these in database too do you? 😅

These are live sources so we will end up with both on the temporal aggregates.

Reply:
I can manually go through the DB entries afterwards to clean them up :)

Reply:
If it's easier we could also do it in the app-backend when we are deduplicating the sources by stripping everything but the domain.